### DISCUSSION TITLE
error with handling errors "cannot read properties of undefined (reading 'endsWith')"

### DISCUSSION THREAD

**Discussion by misha-lysak** [axios/axios#6201] [General]
I want to handle errors from server, but receive an error from latest axios update 
 "cannot read properties of undefined (reading 'endsWith')"
 screenshot added below

![image](https://github.com/axios/axios/assets/63812444/a1511c49-88b0-4a41-9d71-908914915386)

**Comment by robmcguinness** [comment]
Is this running on Node.js or the browser? What runtime version are you using (might not be supported). Just a guess.

**Comment by DigitalBrainJS** [comment]
This issue has been fixed in v1.6.7

**Reply by NavidSHD** [reply]
Did you change yarn version?
